packages/core/src/client.ts

```diff
@@ -3317,7 +3324,7 @@ export class MedplumClient extends TypedEventTarget<MedplumClientEventMap> {
       try {
         body = await response.json();
       } catch (err) {
-        console.error('Error parsing response', response.status, err);
+        this.logger.error(`Error parsing response ${response.status}`, err instanceof Error ? err : undefined);
```

Comment:
I think this is also slightly different with `Logger`, since the properties of error will attempt to be splatted out into the object that is logged